electron/main.ts

```diff
@@ -826,9 +830,19 @@ ipcMain.handle('egsSync', async (event, args) => {
   }
 })
 
-ipcMain.on('addShortcut', async (event, appName) => {
+ipcMain.on('addShortcut', async (event, appName: string, fromMenu: boolean) => {
+  const game = Game.get(appName)
+  game.addShortcuts(fromMenu)
+  openMessageBox({
```

Comment:
do you think it's best to show the dialog only when added manually?

Reply:
If enableDesktopShortcuts or the other is enabled, I think you shouldn't show this.